.github/workflows/python-tox.yml

```diff
@@ -0,0 +1,19 @@
+on: [pull_request, push]
+jobs:
+  build:
+    # Prevent duplicate builds for 'internal' pull requests on existing commits
+    # Credit: https://github.community/t/duplicate-checks-on-push-and-pull-request-simultaneous-event/18012
+    if: github.event.push || github.event.pull_request.head.repo.full_name != github.repository
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: [2.7, 3.5, 3.6, 3.7, 3.8, pypy-2.7, pypy3]
```

Comment:
NB for review: do `pypy-2.7` and `pypy3` respectively correspond to `pypy` and `pypy3` as previously listed in `tox.ini`?